src/components/Summary/index.js

```diff
@@ -7,13 +7,14 @@ import {useImmerReducer} from 'use-immer';
 import { get, post } from 'api';
 import Button from 'components/Button';
 import Card from 'components/Card';
-import Loader from 'components/Loader';
+import ErrorMessage from 'components/ErrorMessage';
 import FormStepSummary from 'components/FormStepSummary';
+import Loader from 'components/Loader';
+import LogoutButton from 'components/LogoutButton';
+import PrivacyCheckbox from 'components/PrivacyCheckbox';
 import { Toolbar, ToolbarList } from 'components/Toolbar';
 import Types from 'types';
 import { flattenComponents } from 'utils';
-import LogoutButton from 'components/LogoutButton';
-import PrivacyCheckbox from '../PrivacyCheckbox';
 
 const PRIVACY_POLICY_ENDPOINT = '/api/v1/config/privacy_policy_info/';
```

Comment:
I think this is my mistake, but there shouldn't be a trailing slash here?

Reply:
yeah and I messed up while rebasing, will fix